plugins/package-managers/bundler/src/main/kotlin/Bundler.kt

```diff
@@ -453,13 +453,17 @@ internal data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val description = sequenceOf(node["description"], node["info"]).mapNotNull { text ->
+                text?.textValue()?.takeIf { it.isNotEmpty() }
```

Comment:
Should this use `textValueOrEmpty()` ?

Reply:
Both would work (as "" is turned to `null` via `takeIf { it.isNotEmpty() }`), but `textValueOrEmpty()` does more than necessary here, so I'd prefer to stick to just `textValue()`.